rust/src/lib.rs

```diff
@@ -2137,12 +2151,12 @@ pub unsafe extern "C" fn zcashlc_propose_shielding(
                 Ok(account_receivers.iter().next().map(|(a, v)| (*a, *v)))
             },
             |addr| Ok(account_receivers.get(&addr).map(|value| (addr, *value)))
-        )?.filter(|(_, value)| *value >= shielding_threshold) {
+        )?.filter(|(_, value)| *value >= shielding_threshold) { Some((addr, _)) => {
             [addr]
-        } else {
+        } _ => {
             // There are no transparent funds to shield; don't create a proposal.
             return Ok(ffi::BoxedSlice::none());
-        };
+        }};
```

Comment:
Was this a clippy suggestion? It seems less clear to me.

Reply:
Rust 2024 edition introduces rustfmt editions, to enable changing or fixing formatting issues. This is one such formatting change in the edition.